packages/run-protocol/src/runStake/runStakeKit.js

```diff
@@ -347,11 +347,14 @@ export const makeRunStakeKit = (zcf, startSeat, manager) => {
     getNotifier: () => state.notifier,
     makeAdjustBalancesInvitation: () => {
       assert(state.open);
-      return zcf.makeInvitation(helper.adjustBalancesHook, 'AdjustBalances');
+      return zcf.makeInvitation(
+        helper.handleAdjustBalancesOffer,
+        'AdjustBalances',
+      );
     },
     makeCloseInvitation: () => {
       assert(state.open);
-      return zcf.makeInvitation(helper.closeHook, 'CloseVault');
+      return zcf.makeInvitation(helper.handleCloseOffer, 'CloseVault');
```

Comment:
This is trying to be consistent with `makeCloseInvitation`. The key is `Close` so the two methods are:
- `makeCloseInvitation`
- `handleCloseOffer`.

Ah, you're probably referring to the string. Yeah I see that I was very inconsistent in that! Working through this, I think that can be more free-form, especially since it's loosely user-facing. We may want to refactor methods names and not have to change that string.